dev/Scroller/APITests/ScrollControllerTests.cs

```diff
@@ -40,6 +41,9 @@ public void SettingScrollControllerProperties()
 
             RunOnUIThread.Execute(() =>
             {
+                // We need the styles of the CompositionScrollController, so lets load it
+                App.AppendResourceDictionaryToMergedDictionaries(App.AdditionStylesXaml);
```

Comment:
If this is needed for every test, you can add a TestInitialize method and put it there (like this - [https://github.com/microsoft/microsoft-ui-xaml/blob/e7e08236a0c28269a8641cfc06adfc3c40766d9f/test/MUXControlsTestApp/Utilities/APITestBase.cs#L38](https://github.com/microsoft/microsoft-ui-xaml/blob/e7e08236a0c28269a8641cfc06adfc3c40766d9f/test/MUXControlsTestApp/Utilities/APITestBase.cs#L38))

Reply:
I thought about that too, however I am not sure if injecting this into every API test, regardless of whether they use them or not is the right behavior here. In my opinion, since there are only a few API tests right now, this seems acceptable to do this on a case by case basis here.
However if you think it is better to load it every time, of course I will adjust the code.